src/main/java/com/ddudu/user/repository/UserRepositoryImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.ddudu.user.repository;
+
+import static com.ddudu.following.domain.QFollowing.following;
+
+import com.ddudu.following.domain.FollowingStatus;
+import com.ddudu.user.domain.User;
+import com.querydsl.core.BooleanBuilder;
+import com.querydsl.jpa.impl.JPAQueryFactory;
+import jakarta.persistence.EntityManager;
+import java.util.List;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class UserRepositoryImpl implements UserRepositoryCustom {
+
+  private final JPAQueryFactory jpaQueryFactory;
+
+  public UserRepositoryImpl(EntityManager em) {
+    this.jpaQueryFactory = new JPAQueryFactory(em);
+  }
+
+  @Override
+  public List<User> findFolloweesOfUser(User follower) {
+    BooleanBuilder whereClause = new BooleanBuilder();
+
+    whereClause.and(following.follower.eq(follower))
+        .and(following.status.eq(FollowingStatus.FOLLOWING));
```

Comment:
```suggestion
    whereClause.and(following.follower.eq(follower))
        .and(following.status.eq(FollowingStatus.FOLLOWING))
        .and(following.isDeleted.eq(false));
```

Following에 `@SQLRestriction("is_deleted = 0")` 가 없으니 isDeleted가 false인 조건을 추가해야 되지 않나요?